src/main/java/universalDiscord/UniversalDiscordPlugin.java

```diff
@@ -64,7 +64,7 @@ public class UniversalDiscordPlugin extends Plugin {
     private final ClueNotifier clueNotifier = new ClueNotifier(this);
 
     private static final Pattern CLUE_SCROLL_REGEX = Pattern.compile("You have completed (?<scrollCount>\\d+) (?<scrollType>\\w+) Treasure Trails\\.");
-    private static final Pattern SLAYER_TASK_REGEX = Pattern.compile("You have completed your task! You killed (?<task>[\\d,]+ [\\w,]+)\\..*");
+    private static final Pattern SLAYER_TASK_REGEX = Pattern.compile("You have completed your task! You killed (?<task>[\\d,]+ [\\w,']+)\\..*", Pattern.UNICODE_CHARACTER_CLASS);
```

Comment:
```suggestion
    private static final Pattern SLAYER_TASK_REGEX = Pattern.compile("You have completed your task! You killed (?<task>[\\d,]+ [^\\.]+)\\..*");
```

Reply:
Suggestion applied in a commit further down